src/app/components/forms/form/helper/terra-form-field.helper.ts

```diff
@@ -36,7 +36,8 @@ export class TerraFormFieldHelper
         categoryPicker:       'category',
         multiCheckBox:        'checkboxGroup',
         noteEditor:           'noteEditor',
-        codeEditor:           'codeEditor'
+        codeEditor:           'codeEditor',
+        slider:               'slider'
```

Comment:
Das ist die legacy map.. Den Slider gibt es in der `TerraDynamicForm` nicht.

Reply:
werde ich ändern 👍